integration/live.yml

```diff
@@ -0,0 +1,56 @@
+---
+storage:
+  dropbox:
+    specs_enabled: true
+    api_token: qlv6P0K5_KAAAAAAAAAwmTZY-GE45tpYiA3gZD1-sn4DlB1qJltrnTpUXC0qo3OG
```

Comment:
@vtm9  Is this your Dropbox API token? You may want to reset it and remove it from this PR before anyone abuses it